src/lib/operators/table_scan.hpp

```diff
@@ -48,6 +48,28 @@ class TableScan : public AbstractReadOnlyOperator {
    */
   std::vector<ChunkID> excluded_chunk_ids;
 
+  struct TableScanPerformanceData : public OperatorPerformanceData<AbstractOperatorPerformanceData::NoSteps> {
```

Comment:
Naming: As this is already part of the TableScan class, just "PerformanceData"?

Reply:
I would be fine with that name, but we would have several structs with the same name in various operators which I didn't like.
Your call.

Reply:
As their use outside of the class requires the class name as a qualifier, I don't see an issue here. No strong feelings though.

Reply:
Done for `IndexJoinPerformanceData` and `TableScanPerformanceData`.